app/Http/Kernel.php

```diff
@@ -55,6 +55,7 @@ class Kernel extends HttpKernel
      */
     protected $routeMiddleware = [
         'auth' => \App\Http\Middleware\Authenticate::class,
+        'AuthCheck' => \App\Http\Middleware\AuthCheck::class,
```

Comment:
```suggestion
        'auth.check' => \App\Http\Middleware\AuthCheck::class,
```
array keys ids for routes middleware must be smallcase and dot(.) separated.